protostar/protostar_toml/protostar_toml_section.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+from typing import Dict, List, Union
+
+
+class ProtostarTOMLSection(ABC):
+    Dict = Dict[str, Union[str, int, bool, List[str]]]
```

Comment:
This is cryptic + i suggest using different name than in standard typing to avoid being misleading,